evm/src/fuzz/strategies/param.rs

```diff
@@ -25,22 +25,24 @@ pub fn fuzz_param(param: &ParamType) -> impl Strategy<Value = Token> {
         // rejection sampling if it's determined the bias is too severe. Rejection sampling may
         // slow down tests as it resamples bad values, so may want to benchmark the performance
         // hit and weigh that against the current bias before implementing
-        ParamType::Int(n) => match n / 8 {
-            32 => any::<[u8; 32]>()
-                .prop_map(move |x| I256::from_raw(U256::from(&x)).into_token())
-                .boxed(),
-            y @ 1..=31 => any::<[u8; 32]>()
+        ParamType::Int(n) => {
+            // use uint strat but convert to int
+            let nc = *n;
+            super::UintStrategy::new(*n, vec![])
```

Comment:
Let's also do the same thing for `FixedBytes` in this PR, since the strategy for `bytesN` types pretty much be identical to the strategy for `uintN` types